### DISCUSSION TITLE
[Potential Bug] Booster UI log-trace association query

### DISCUSSION THREAD

**Discussion by Superskyyy** [apache/skywalking#10338] [General]
In the General Services tab, when I clicked on the `view log` button related to a specific Trace (call this Action 1). The UI seems to have the TraceID query field remembered in the Log tab regardless of how many further actions were carried out, such as doing a manual search, removing the TraceID from the field etc. When switching away and then back to the Log view, the last TraceID from Action 1 will always reappear. This doesn't sound like correct behaviour (although I don't know how to fix it, my Vue knowledge is very limited)

To reproduce:
1. Click on any trace that has a log associated, and click `view logs`
2. After landing on the logs tab, the TraceID query field is set to be the TraceID in step1
3. Do some other queries on the logs tab, or simply remove the field to view all logs. (This step can be done infinite times)
4. Revisit the log tab after switching away; the traceID from step 1 reappears.

I think the better behaviour will be either to keep the real last query in the logs tab OR reset back to an initial state (empty the TID field).

![image](https://user-images.githubusercontent.com/26076517/216737221-7f0eb8c1-e458-496b-8829-d841c0cf7ae9.png)

**Comment by wu-sheng** [comment]
@Fine0830 Could you take a look?

**Reply by wu-sheng** [reply]
We should reset all conditions before entering the log widget. In case of conditions mixed or/and dirty conditions left.

**Comment by Fine0830** [comment]
I can reproduce it and submit an [issue](https://github.com/apache/skywalking/issues/10340) to follow this.
